src/verification.rs

```diff
@@ -982,13 +948,19 @@ impl VerificationRequest {
     /// supports the given verification methods.
     ///
     /// `methods` represents the methods that we should advertise as
-    /// supported by us.
+    /// supported by us. Items inside `methods` are going to be deleted on the
+    /// JS part after the function returns.
+    ///
+    /// Items inside `methods` will be invalidated by this method.
```

Comment:
We're saying the same thing twice here:

```suggestion
    /// supported by us.
    ///
    /// Items inside `methods` will be invalidated by this method.
```